app/src/main/java/org/oppia/app/player/state/StateAdapter.kt

```diff
@@ -121,4 +141,26 @@ class StateAdapter(
       binding.executePendingBindings()
     }
   }
+
+  inner class SelectionInteractionViewHolder(
+    val binding: ViewDataBinding
+  ) : RecyclerView.ViewHolder(binding.root) {
+    internal fun bind(customizationArgs: CustomizationArgsInteractionViewModel) {
+      val items: Array<String>?
+      val choiceContentList: MutableList<SelectionContentViewModel> = ArrayList()
+      binding.executePendingBindings()
```

Comment:
Why execute bindings here? Nothing seems to have changed yet.

Reply:
Removed